src/ignore_me.rs

```diff
@@ -75,8 +75,12 @@ fn unwrap_path(it: Result<ignore::DirEntry, ignore::Error>) -> PathBuf {
     it.expect("Found invalid path: Aborting").into_path()
 }
 
-fn remove_dir_files(files: &[PathBuf], verbose: bool) {
-    for item in files.iter().filter(|file| file.exists()) {
+pub fn remove_dir_files<T: IntoIterator<Item = I>, I: Into<PathBuf>>(files: T, verbose: bool) {
```

Comment:
suggestion: working with the `impl` syntax helps to improve the readability quite a lot.

so here we would end up having this:
```rust
pub fn remove_dir_files(files: impl IntoIterator<Item = impl Into<PathBuf>>, verbose: bool) {
```

But of course this is quite matter of taste, but this syntactical sugar was introduced for a reason right? :)